src/bp-templates/bp-nouveau/buddypress/search/loop/post-ajax.php

```diff
@@ -7,7 +7,7 @@
 			<img
 				src="<?php echo $result['post_thumbnail']; ?>"
 				class="attachment-post-thumbnail size-post-thumbnail wp-post-image"
-				alt="<?php the_title() ?>"
+				alt="<?php esc_attr( the_title() ); ?>"
```

Comment:
@ngolwalkar `the_title()` will print directly so `esc_attr` will not work. You should use `echo esc_attr( get_the_title() );` 

also `the_title()` no need to escape as is already escaping by WordPress